pkg/security/module/module.go

```diff
@@ -229,18 +243,42 @@ func (m *Module) Reload() error {
 
 	ruleSet := m.probe.NewRuleSet(newRuleSetOpts())
 
-	loadErr := rules.LoadPolicies(m.config.PoliciesDir, ruleSet)
+	loadErr := rules.LoadPolicies(policiesDir, ruleSet)
 
 	model := &model.Model{}
 	approverRuleSet := rules.NewRuleSet(model, model.NewEvent, newRuleSetOpts())
-	loadApproversErr := rules.LoadPolicies(m.config.PoliciesDir, approverRuleSet)
+	loadApproversErr := rules.LoadPolicies(policiesDir, approverRuleSet)
 
 	if loadErr.ErrorOrNil() != nil {
 		logMultiErrors("error while loading policies: %+v", loadErr)
 	} else if loadApproversErr.ErrorOrNil() != nil {
 		logMultiErrors("error while loading policies for Approvers: %+v", loadApproversErr)
 	}
 
+	monitor := m.probe.GetMonitor()
+	ruleSetLoadedReport := monitor.PrepareRuleSetLoadedReport(ruleSet, loadErr)
+
+	if selfTestPolicy != nil {
```

Comment:
Sorry kind of a nit again, but this could help keep the "readWithPoliciesDir" function short and sweet: could you maybe put everything below this if into a method of `m.selfTester` ?